server/src/main/java/org/apache/druid/discovery/NodeType.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.discovery;
+
+/**
+ * Constant names are lowercase for backward compatibility, as a part of the {@link DiscoveryDruidNode}'s JSON format.
+ */
+public enum NodeType
+{
+  coordinator,
```

Comment:
I think all these enum values in uppercase would be better.

Reply:
You can customize this with a `@JsonValue` annotation: see LongEncodingStrategy for an example.

Reply:
Right, changed.